test/files/setup_databases.jl

```diff
@@ -35,24 +37,32 @@ function parse_commandline()
         "--output", "-o"
         help = "path where the database folder is going to be created."
         default = "."
-        "--pdb"
+        """
+        --pdb
+        """
```

Comment:
I personally don't mind this and I don't think there's an easy way to get around this.

Reply:
I thought about changing it to do oneliners with single quotes, but I think I like this better. It makes it easier to add more to the docstring later on